components/Buttons/MarkWatchedButton.vue

```diff
@@ -0,0 +1,72 @@
+<template>
+  <v-btn v-if="canMarkWatched" icon dark @click.stop.prevent="togglePlayed">
```

Comment:
Why the `.stop.prevent`?

Reply:
Without it, if you click on the icon, it redirect to the item details page instead of liking the item. Stop.prevent.... Is also on item menu and like button

Reply:
Ok nice, though that's weird cause in my previous experiences with Vuetify I never did that